data/core/units/nagas/Zephyr.cfg

```diff
@@ -0,0 +1,97 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Naga Zephyr
+    name= _ "Naga Zephyr"
+    race=naga
+    gender=male
+    image="units/nagas/zephyr.png"
+    profile=portraits/nagas/fighter.png
+    hitpoints=53
+    movement_type=naga
+    movement=7
+    experience=100
+    level=3
+    alignment=liminal
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=50
+    usage=fighter
+    description= _ "Description by Tom_Of_Wesnoth
+
+    Those Naga who master the use of their ancestral weapon, the blade ring, are known among their people as Zephyr, and are held in the greatest of respect as warriors. Amongst the neighbouring Dunefolk, the Zephyr are coveted as mercenaries, with their mastery of a foreign and ill-understood weapon providing them with a significant advantage in combat. Zephyr wield their blade rings with confidence and familiarity, whirling the wicked blades around their fingers as fast as the naked eye can behold, and throwing them faster still. Zephyr give their foes no respite, using both hands to unleash an unending storm of blade rings upon them.
+    
+    
+    Idea by ghype
+    
+    As to the Ringcaster and Zephyr, they could be renegade sell swords who still work on paid duties but settled down in the more tribe-ish parts of the DF lands and do more odd jobs for local villagers (renegade opposed to the life of sell-swords).
+    "
+    die_sound=naga-die.ogg
+    {DEFENSE_ANIM "units/nagas/zephyr.png" "units/nagas/zephyr.png" {SOUND_LIST:NAGA_HIT} }
+
+    [defense]
+        flat=50
+        sand=40
+    [/defense]
+
+    [attack]
+        name=moonglaive
+        description=_"moonglaive"
+        type=blade
+        range=melee
+        damage=6
+        number=3
+        icon=attacks/moonglaive.png
+    [/attack]
```

Comment:
The problem is for all nagas, their weapon name should be "curved blade" and portrait "blade-curved". For Bladewhirler and Dervish there are good weapon names.